item-spec/json-schema/common.json

```diff
@@ -8,9 +8,15 @@
     {
       "$ref": "basics.json"
     },
+    {
+      "$ref": "bands.json"
+    },
```

Comment:
Isn't there a loop dependency here? Band is only part of the asset object right?

Reply:
No, but it is recursive, indeed.